napari/_qt/qt_event_loop.py

```diff
@@ -289,21 +298,5 @@ def run(
         )
         return
 
-    with _install_hooks(gui_exceptions):
+    with notification_manager:
```

Comment:
if I do it this way , the ExceptionHandler does not seem needed anymore; should I remove it and all references to it?

Reply:
In particular that will remove NAPARI_CATCH_ERRORS and NAPARI_EXIT_ON_ERROR, but I can restore those in the notification manager.